translator/monadic/examples/arraySearchProgScript.sml

```diff
@@ -5,7 +5,7 @@
 
 open preamble ml_monad_translator_interfaceLib
 
-val _ = new_theory "array_searchProg"
+val _ = new_theory "arraySearchProg"
```

Comment:
This is a break away from conventions. Why rename?

Reply:
I'll change some file names under `translator/monadic/examples` to be more inline with file name conventions.